src/Resources/app/storefront/src/snippets/de_DE/finsearch.de_DE.json

```diff
@@ -9,6 +9,7 @@
     "queryInfoMessage": {
       "filter": "Suchergebnisse für %filterName% <strong>%filterValue%</strong> (<strong>%hits%</strong> Treffer)",
       "query": "Suchergebnisse für <strong>%query%</strong> (<strong>%hits%</strong> Treffer)",
+      "shoppingGuide": "Suchergebnisse für <strong>%wizard%</strong> (<strong>%hits%</strong> Treffer)",
```

Comment:
Please rename `%wizard%` to something, that is easier to understand like `%shoppingGuideName%` or maybe even just `%shoppingGuide%`. This is just internal naming and should not be used for public translations, etc.